pkg/utils/validation/cidr/cidr_test.go

```diff
@@ -44,15 +49,15 @@ var _ = Describe("cidr", func() {
 
 		})
 
-		Context("GetCIDR", func() {
+		Describe("GetCIDR", func() {
```

Comment:
Why did you change `Context` to `Describe` consistently?

Reply:
Technically, both are the same.
Semantically, however, `Describe` is commonly used to specify which code is tested and `Context` in which situation/scenario it is tested: also see https://onsi.github.io/ginkgo/#organizing-specs-with-container-nodes
I cleaned it up in this test file along the way.